app/client/cli/servicer.go

```diff
@@ -0,0 +1,243 @@
+}
+
+func NewServicerCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "Servicer",
+		Short:   "Servicer specific commands",
+		Aliases: []string{"servicer"},
+		Args:    cobra.ExactArgs(0),
+	}
+
+	cmds := servicerCommands()
+	applySubcommandOptions(cmds, attachPwdFlagToSubcommands())
+	applySubcommandOptions(cmds, attachKeybaseFlagsToSubcommands())
+
+	cmd.AddCommand(cmds...)
+
+	return cmd
+}
+
+func servicerCommands() []*cobra.Command {
+	cmdDef := actorCmdDef{"Servicer", coreTypes.ActorType_ACTOR_TYPE_SERVICER}
+	cmds := []*cobra.Command{
+		newStakeCmd(cmdDef),
+		newEditStakeCmd(cmdDef),
+		newUnstakeCmd(cmdDef),
+		newUnpauseCmd(cmdDef),
+		{
+			// IMPROVE: allow reading the relay payload from a file with the serialized protobuf via [--input_file]
+			Use:   "Relay <applicationAddrHex> <servicerAddrHex> <relayChainID> <relayPayload>",
+			Short: "Relay <applicationAddrHex> <servicerAddrHex> <relayChainID> <relayPayload>",
+			Long: `Sends a trustless relay using <relayPayload> as contents, to the specified active <servicerAddrHex> in the the <applicationAddrHex>'s session.
+Will prompt the user for the *application* account passphrase`,
+			Aliases: []string{},
+			Args:    cobra.ExactArgs(4),
+			RunE: func(cmd *cobra.Command, args []string) error {
+				applicationAddr := args[0]
+				servicerAddr := args[1]
+				chain := args[2]
+				relayPayload := args[3]
+
+				// REFACTOR: decouple the client logic from the CLI
+				//	The client will: send the trustless relay and return the response (using a single function as entrypoint)
+				//	The CLI will:
+				//		1) extract the required input from the command arguments
+				//		2) call the client function (with the inputs above) that performs the trustless relay
+				pk, err := getPrivateKeyFromKeybase(applicationAddr)
+				if err != nil {
+					return fmt.Errorf("error getting application's private key: %w", err)
+				}
+
+				// TECHDEBT(#791): cache session data
+				session, err := getCurrentSession(cmd.Context(), applicationAddr, chain)
+				if err != nil {
+					return fmt.Errorf("Error getting current session: %w", err)
+				}
+
+				servicer, err := validateServicer(cmd.Context(), session, servicerAddr)
+				if err != nil {
+					return fmt.Errorf("error getting servicer for the relay: %w", err)
+				}
+
+				relay, err := buildRelay(relayPayload, pk, session, servicer)
+				if err != nil {
+					return fmt.Errorf("error building relay from payload: %w", err)
+				}
+
+				fmt.Printf("sending trustless relay for %s to %s with payload: %s\n", applicationAddr, servicerAddr, relayPayload)
+
+				resp, err := sendTrustlessRelay(cmd.Context(), servicer.ServiceUrl, relay)
+				if err != nil {
+					return err
+				}
+
+				fmt.Printf("HTTP status code: %d\n", resp.HTTPResponse.StatusCode)
+				fmt.Println("Response: ", resp.JSON200)
+
+				return nil
+			},
+		},
+	}
+
+	return cmds
+}
+
+// TODO: add a cli command for fetching sessions
```

Comment:
I think I already asked this but why is `/v1/client/get_session:` insufficient?

Reply:
Yes, there was a [review comment](https://github.com/pokt-network/pocket/pull/778#discussion_r1210830146) on this. Technically the `/v1/client/get_session` is sufficient in terms of actually having access to the functionality. Like the rest of the CLI, it is a matter ease of use. Considering we will be addressing #791 soon, i.e. there will be a client-side cache for sessions, adding a command for fetching sessions makes sense IMO.